examples/tests/Earthfile

```diff
@@ -224,7 +224,7 @@ remote-test:
     RUN --privileged \
         --entrypoint \
         --mount=type=tmpfs,target=/tmp/earthly \
-        -- --no-output github.com/earthly/test-remote/builtin-args:main+test
+        -- --no-output github.com/earthly/test-remote/builtin-args:vlad/add-ts-tests+test
```

Comment:
will this get backed out once https://github.com/earthly/test-remote/pull/1#pullrequestreview-727702130 gets merged?

Reply:
yes